File: DACAug2006/Java/Core/Threads/ThreadTest1.java
class ThreadTest1{
	
	public static void main(String[] args){
		Runnable r = new Runnable(){
			public void run(){
				for(int i = 1; i <= 3000; i++){
					System.out.println("Hello : " + i);
				}
					
			}
		};
		Thread t = new Thread(r);
		t.start();
		Thread tt = new Thread(r);
		tt.start();
		for(int i = 1; i <= 2000; i++){
			System.out.println("Welcome : " + i);
		}
		
	}
}

File: DACAug2006/VC++/Win32/KernelAPI/Threads/threadtest1.c
#include <stdio.h>
#define UNICODE
#include <windows.h>

DWORD WINAPI ThreadProc(LPVOID lpParam)
{
	int i;

	for(i = 1; i < 3001; i++)
		wprintf(L"Hello %d from thread with ID %d\n", i, GetCurrentThreadId());
	return 0;
}

int wmain()
{
	int i;
	HANDLE hThread = CreateThread(NULL, 0, ThreadProc, NULL, 0, NULL);
	for(i = 1; i < 2001; i++)
		wprintf(L"Welcome %d from thread with ID %d\n", i, GetCurrentThreadId());
	WaitForSingleObject(hThread, INFINITE);
	CloseHandle(hThread);
}








